benchmarks/core.py

```diff
@@ -6,12 +6,15 @@
 
 
 class BaseBenchmark:
+    def __init__(self, name):
+        self.name = name
+
```

Comment:
Small nitpick to prefer private attributes:
```suggestion
    def __init__(self, name):
        self._name = name
```